src/EntityComponentManager.cc

```diff
@@ -772,6 +772,9 @@ void EntityComponentManagerPrivate::SetRemovedComponentsMsgs(Entity &_entity,
 
   // Find the entity in the message
   auto entIter = _msg.mutable_entities()->find(_entity);
+  if (entIter == _msg.mutable_entities()->end()) {
```

Comment:
FYI, https://github.com/ignitionrobotics/ign-gazebo/pull/495 will be changing this.

Reply:
Ok cool. I was worried it was a concern with nested models I hadn't taken into account. I saw regular segfaults here. Should we submit this as is or wait for the new commit to be submitted and forward-ported?

Reply:
If the segfaults were regular enough to prevent development/testing, then let's submit it like this. Otherwise, it would be nice to avoid a merge conflict.

Reply:
It crashes pretty easily without this check